example/test_runner.dart

```diff
@@ -20,49 +20,53 @@ main() {
       abbr: 'c',
       defaultsTo: 'none',
       help: 'Specify any compilation step (if needed).',
-      allowed: ['none', 'dart2js', 'dartc'],
+      allowed: [
+        'none',
+        'dart2js',
+        'dartc'
```

Comment:
dartfmt's behavior can be subtle in some cases. If I recall, there is a rule that says that if all of the trailing arguments of an argument lists are collection literals, then if any of them split (like the allowedHelp map below), then they all should. It avoids weird cases like:

```dart
function(first: [
  "stuff",
  "more",
  "stuff",
  "more",
  "stuff",
  "more",
  ], second: ["short"], third: [
  "stuff",
  "more",
  "stuff",
  "more",
  "stuff",
  "more",
])
```